### ISSUE TITLE
Show matching jobs (for job seekers) to job developer/case manager

### ISSUE THREAD

**Issue by patmbolger** [AgileVentures/MetPlus_tracker#388]
As a job developer
I want to see jobs that match the qualifications of each of the job seekers assigned to me

**Comment by patmbolger** [comment]
This is gated by #332.

The story includes:

- [ ] In the job developer (agency person) home page, add a column ("Jobs") to the table "Your Job Seekers (as job developer)".  This column contains the count of jobs that match the job seekers qualifications.

- [ ] The count column above is a link to a view listing the jobs.  The title of each such job is a link to the job *show* view (where theJD can submit an application on behalf of the job seeker (separate story))

**Comment by patmbolger** [comment]
This is gated by #419

**Comment by patmbolger** [comment]
For development purposes you should use the 'sandbox' API for the Cruncher:

1. You'll have to set the cruncher service URL to point to the sandbox rather than the production version of the cruncher.  To do this, in `config/application.yml`, change the version indicator ('v1' or 'v2') for CRUNCHER_SERVICE_URL to 'v99999'.  (the rest of the URL should be unchanged).

2. The cruncher API will now return canned responses for job and résumé matching.  The expected results are described here: https://github.com/AgileVentures/MetPlus_PETS/wiki/Cruncher-Dev-Test-Sandbox

The actual mechanics of invoking the API are taken care of for the developer - you will just use methods in the ResumeCruncher and JobCruncher classes.
